src/framework/core/js/ModelTransformationTransforms.js

```diff
@@ -653,12 +695,11 @@ var fluid = fluid || fluid_2_0_0;
         return parsedOffset;
     };
 
-    fluid.transforms.invertArrayIndexation = function (transformSpec, transformer) {
-        var togo = fluid.model.transform.copyInversePaths(transformSpec, transformer);
-        if (!isNaN(Number(togo.offset))) {
-            togo.offset = Number(togo.offset) * (-1);
+    fluid.transforms.invertArrayIndexation = function (transformSpec) {
```

Comment:
Hi @kaspermarkus - I think this then shows up a hole in our "type system" since hopefully it is not completely accidental that linearScale uses an input named "input" and uses it in exactly the same sense as a standardInputTransform. What I propose is that to signal this fact we then turn linearScale into something which inherits *both* from standardInputTransform and multiInputTransform (together with suitable support in the core transformer to recognise what this means) which then would hopefully mean that we can get rid of the unsightly manual call to copyInputPaths in its body.